client/crypt.c

```diff
@@ -29,15 +29,9 @@ extern unsigned int xOPENSSL_ia32cap_P[4];
 extern int xOPENSSL_ia32_cpuid(unsigned int *);
 
 // initialization of the encryption system
-int xdag_crypt_init(int withrandom)
```

Comment:
Not sure what is `withrandom` used for.

Reply:
if `withrandom` is not zero, OpenSSL random generator was seeded by some random array generated based on dnet key. 
Function out_balances was calling xdag_crypt_init with `withrandom` = 0